components/Chat/Message/MessageReactionsList.tsx

```diff
@@ -210,14 +212,20 @@ const MessageReactionsListInner: FC<MessageReactionsListProps> = ({
   return (
     <View style={styles.container}>
       {list.length !== 0 ? (
-        <FlatList
-          contentContainerStyle={styles.reactionsContainer}
-          data={list}
-          horizontal
-          renderItem={renderItem}
-          keyExtractor={keyExtractor}
-          showsHorizontalScrollIndicator={false}
-        />
+        <Portal>
```

Comment:
I don't understand the use of Portal here. Isn't Portal used to render somewhere else but it needs a PortalHost?

Reply:
@saulmc 
Allright not sure I know how this works
Documentation at https://callstack.github.io/react-native-paper/docs/components/Portal/ says
> It requires a [Portal.Host](https://callstack.github.io/react-native-paper/docs/components/Portal/PortalHost) component to be rendered somewhere in the parent tree
and by checking out your branch I don't see PortalHost anywhere
Am I misunderstanding the doc?

Reply:
I think it is included in the PortalProvider in App.tsx. From the docs:

> Note that if you're using the Provider component, this already includes a Portal.Host.

Reply:
Also, I don't really know how this works either -- just copying a pattern that Alex had used for the message bubble.

Reply:
got it. Let's go then!